<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #f2e6cf;
    border: 1px solid #f2e6cf; 
    font: 15pt &quot;Futura&quot;;
}

QFrame {
    background-color: #9cd2d3;
    border-radius: 10px;
    border: 1px solid #f2e6cf; 
}

QPushButton {
    font: 57 20pt &quot;Futura&quot;;
    color: rgb(0, 0, 0);
    background-color: #f2e6cf;
    border-radius: 6px;  
    border: 1px solid #f2e6cf; 
}

QPushButton:hover {
    background-color: rgb(7, 153, 182); 
}

QLabel {
    font: 15pt &quot;Futura&quot;; 

}

QLineEdit {
    background-color: #f2e6cf;
    border-radius: 10px;
    border: 1px solid #9cd2d3; 
}


</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,4">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_botones">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="bt_menu">
            <property name="text">
             <string>Menu</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../menu.svg.png</normaloff>../menu.svg.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>34</width>
              <height>34</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_menos">
            <property name="text">
             <string>--</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../minimizar.svg.png</normaloff>../minimizar.svg.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>34</width>
              <height>36</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_minimizar">
            <property name="text">
             <string>Minimizar</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../minim.svg.png</normaloff>../minim.svg.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_maximizar">
            <property name="text">
             <string>Maximizar</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../maxim.svg.png</normaloff>../maxim.svg.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_cerrar">
            <property name="text">
             <string>X</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../cerrar.svg.png</normaloff>../cerrar.svg.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_contenidos">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
          <item>
           <widget class="QFrame" name="frame_menu">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="botondatos">
               <property name="text">
                <string>Datos</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="botonconectar">
               <property name="text">
                <string>Conectar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="botonver">
               <property name="text">
                <string>Ver</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_graficos">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QStackedWidget" name="stackedWidget">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="page_graf">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="GraphicsLayoutWidget" name="grafico"/>
                 </item>
                 <item>
                  <layout class="QGridLayout" name="gridLayout">
                   <item row="0" column="1">
                    <widget class="QLabel" name="label_bpm">
                     <property name="text">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QPushButton" name="botoniniciar">
                     <property name="text">
                      <string>Iniciar</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QPushButton" name="botonpausa">
                     <property name="text">
                      <string>Pausa</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QPushButton" name="botonguardar">
                     <property name="text">
                      <string>Guardar</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QPushButton" name="botonclear">
                     <property name="text">
                      <string>Clear</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>                        BPM</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_datos">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <layout class="QGridLayout" name="gridLayout_2">
                   <item row="0" column="0">
                    <widget class="QLabel" name="labelnombre">
                     <property name="font">
                      <font>
                       <family>Futura</family>
                       <pointsize>15</pointsize>
                       <weight>50</weight>
                       <italic>false</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>  Nombre </string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="editnombre"/>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="labelsexo">
                     <property name="font">
                      <font>
                       <family>Futura</family>
                       <pointsize>15</pointsize>
                       <weight>50</weight>
                       <italic>false</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>  Sexo </string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QComboBox" name="sexobox">
                     <item>
                      <property name="text">
                       <string>Masculino</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Femenino</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="labeledad">
                     <property name="font">
                      <font>
                       <family>Futura</family>
                       <pointsize>15</pointsize>
                       <weight>50</weight>
                       <italic>false</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>  Edad </string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="editedad"/>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="labelfechanac">
                     <property name="font">
                      <font>
                       <family>Futura</family>
                       <pointsize>15</pointsize>
                       <weight>50</weight>
                       <italic>false</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>  Fecha de nacimiento </string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1">
                    <widget class="QDateEdit" name="fechanac"/>
                   </item>
                   <item row="4" column="0">
                    <widget class="QLabel" name="labelcomentarios">
                     <property name="font">
                      <font>
                       <family>Futura</family>
                       <pointsize>15</pointsize>
                       <weight>50</weight>
                       <italic>false</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>  Comentarios </string>
                     </property>
                    </widget>
                   </item>
                   <item row="4" column="1">
                    <widget class="QLineEdit" name="editcomentarios"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
